### ISSUE TITLE
NTR: intraosseous lipoma

### ISSUE THREAD

**Issue by peupeubangbang** [obophenotype/human-phenotype-ontology#10653]
**Preferred term label:**
Intraosseous lipoma

**Synonyms**
Bone lipoma, bony lipoma, lipoma of bone, intra-osseous lipoma, intra-bony lipoma

**Definition (free text, please give PubMed ID)**
Intraosseous lipomas are rare benign lesions composed of mature fat that reside within the intramedullary canal.

**Comment**
They account for an estimated 0.1% of primary bone tumours. No age or sex predilection has been reported. These tumours are often asymptomatic and generally are discovered incidentally on imaging, particularly MRI. Symptomatic intraosseous lipomas typically present with pain, local swelling, or tenderness. The metaphysis of long bones and the calcaneus are the most common locations.

PMID:27298943

**Parent term (use hpo.jax.org/app)**
HP:0012032 Lipoma

**Comment by pnrobinson** [comment]
added!, thanks
